wodles/azure/azure-logs.py

```diff
@@ -828,9 +828,16 @@ def send_message(message: str):
         The message body to send to analysisd.
     """
     s = socket(AF_UNIX, SOCK_DGRAM)
+
+    encoded_msg = f'{SOCKET_HEADER}{message}'.encode(errors='replace')
+
+    # Logs warning if event is bigger than max size
+    if len(encoded_msg) > MAX_EVENT_SIZE:
+        logging.warning(f"WARNING: Event size exceeds the maximum allowed limit of {MAX_EVENT_SIZE} bytes.")
```

Comment:
You may remove `WARNING` from the log message since the logger already adds it.

Reply:
Fixed in commit 113469afbaff8582af4db430587402c60c697277:

https://github.com/wazuh/wazuh/blob/113469afbaff8582af4db430587402c60c697277/wodles/aws/aws_s3.py#L519-L520